microprofile/WFLY-17156-MicroProfile-Telemetry.adoc

```diff
@@ -0,0 +1,117 @@
+= WFLY-17156 - MicroProfile Telemetry
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+MicroProfile 6 introduces a new specification, MicroProfile Telemetry. This spec replaces MicroProfile OpenTracing in the platform.
+
+This RFE will add support for this new specification as part of the MicroProfile configurations.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17156[WFLY-17156 - Implement Support for MicroProfile Telemetry]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1976[EAP7-1976 - Implement Support for MicroProfile Telemetry]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+
+* [x] Engineering
+* [ ] QE
+
+=== Affected Projects or Components
+The OpenTelemetry support in standard WildFly will be modified as need so as to share as much code as possible between the closely-related modules.
+
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [ ] Managed domain
+* [x] OpenShift s2i
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+The full MicroProfile Telemetry spec must be supported. This includes:
+
+* Automatic instrumentation of REST and REST client requests
+* Manual instrumentation via the `@WithSpan` annotation
+* The injection of various OpenTelemetry APIs:
+** `OpenTelemetry`
+** `Tracer`
+** `Span`
+** `Baggage`
+* Remove the MicroProfile OpenTracing extension from the standard configurations (see `Interoperability` below).
+
```

Comment:
I think the Requirements section should cover the relationship to the OpenTelemetry subsystem.

Reply:
Updated.